From 2f2ef3473451dd56cf7a99dd12a803a6dcfb6d9d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Mon, 12 Dec 2011 13:22:44 +0200
Subject: [PATCH 12/13] Install public headers

The path should be $INCLUDEPATH/Qt3D - at least that path is referenced
in code using QtQuick3D classes.
---
 src/threed/threed.pro |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/src/threed/threed.pro b/src/threed/threed.pro
index cbc88ef..7e29900 100644
--- a/src/threed/threed.pro
+++ b/src/threed/threed.pro
@@ -24,3 +24,8 @@ HEADERS += $$PRIVATE_HEADERS
 DEFINES += QT_BUILD_QT3D_LIB
 
 !contains(QT_CONFIG, egl):DEFINES += QT_NO_EGL
+
+devel.files = $$PUBLIC_HEADERS
+devel.path = $$[QT_INSTALL_HEADERS]/$$TARGET
+
+INSTALLS += devel
-- 
1.7.7.3

